pkg/client/api_client.go

```diff
@@ -261,12 +271,14 @@ func (c *APIClient) findTarget(target *api.Target) (*api.Target, error) {
 	// by comparing the category, target type and identifier fields
 	// target type is regarded the same if the old one only differs by an extra suffix
 	for _, tgt := range targetList {
+		c.printTarget("Trying to match with target", &tgt)
 		// array of InputFields
 		for _, inputField := range tgt.InputFields {
 			if inputField.Name == "targetIdentifier" &&
 				inputField.Value == targetId &&
 				tgt.Category == target.Category &&
 				strings.HasPrefix(tgt.Type, target.Type) {
+				glog.V(4).Infof("Found target match")
 				return &tgt, nil
 			}
 		}
```

Comment:
Should we also log when we do not find a match?

Reply:
We already doing it:

```
glog.V(4).Infof("target %v does not exist", targetId)
```